src/js/component/recorder.vue

```diff
@@ -3,35 +3,29 @@
         v-bind:class="{
             'recorder--recording': isRecording
         }"
-        v-if="isSupported">
```

Comment:
A `v-if` on the root node of a component is an anti pattern, and should be use on the parent.